applications/GeoMechanicsApplication/custom_elements/geo_steady_state_Pw_piping_element.h

```diff
@@ -241,7 +318,7 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) GeoSteadyStatePwPipingElement : publ
         return result;
     }
 
-    Matrix FillPermeabilityMatrix(double pipe_height) const
+    static Matrix FillPermeabilityMatrix(double pipe_height)
     {
         return ScalarMatrix{1, 1, std::pow(pipe_height, 3) / 12.0};
```

Comment:
Just for my knowledge, it should be power of 4 multiplied by a pipe factor, does it mean the factor is dimensional now? Thank you.

Reply:
Apparently yes, Jon and myself share your questioning.